.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 
 go:
   - 1.9
-  - 1.10
```

Comment:
I've seen this issue before. Fix can be found [here](https://github.com/travis-ci/travis-ci/issues/9247)